### ISSUE TITLE
Small understandability improvement in Pytorch basics

### ISSUE THREAD

**Issue by dvsrepo** [yunjey/pytorch-tutorial#3]
Hi!

First of all, really nice resource for learning pytorch and neural nets!

I am following your tutorial for preparing a short hands-on tutorial

During this process, I have identified a very small understandability issue here https://github.com/yunjey/pytorch-tutorial/blob/master/tutorials/00%20-%20PyTorch%20Basics/main.py#L74

If understand correctly, your intention is to show how the loss goes down after the first opt step, but you do not make a new forward pass with the new weights, thus the loss is the same as previously. Making linear(x) either in the call or more explicitly reassigning pred = linear(x) would help to show how the loss is reduced. As I said, it is a minor issue but thought it might help!

Thank you!

**Comment by yunjey** [comment]
Thanks, you're right. I fixed the code.
